testplan/testing/multitest/driver/app.py

```diff
@@ -411,13 +397,20 @@ def stopping(self) -> None:
             )
             raise RuntimeError(err_msg)
 
-    def _make_dirs(self) -> None:
+    def make_runpath_dirs(self) -> None:
+        """
+        Create mandatory directories and install files from given templates
+        using the drivers context before starting the application binary.
+        """
+        super(App, self).make_runpath_dirs()
+
         bin_dir = os.path.join(self.runpath, "bin")
         etc_dir = os.path.join(self.runpath, "etc")
-        for directory in (bin_dir, etc_dir):
+        for directory in (bin_dir, etc_dir, self.app_path):
             makedirs(directory)
         self._binpath = bin_dir
         self._etcpath = etc_dir
+        self.std = StdFiles(self.app_path)
```

Comment:
Std files are not dirs,  it might could go back to pre_start

Reply:
That is true, but we have a circular dependency here, because we need self.std to be defined for install_files() to work properly and also cannot be executed before make_runpath_dirs() as it needs the directories being created here.
So another solution would be to rename make_runpath_dirs function to a better describing name, but does it worth the hassle?